.github/workflows/_lambda-do-release-runners.yml

```diff
@@ -80,9 +80,18 @@ jobs:
           tag: ${{ inputs.tag }}
           updateOnlyUnreleased: true
 
-  # lambda/ci-queue-pct
-  release-ci-queue-pct:
-    name: Upload Release for ci-queue-pct lambda
+  release-python-lambdas:
+    strategy:
+      fail-fast: false
+      matrix:
+        include: [
+          { dir-name: 'ci-queue-pct',               zip-name: 'ci-queue-pct' },
+          { dir-name: 'oss_ci_job_queue_time',      zip-name: 'oss-ci-job-queue-time' },
+          { dir-name: 'oss_ci_cur',                 zip-name: 'oss-ci-cur' },
```

Comment:
Should we rename the 2 folders above to match the zip-name since all the other lambda's do? this would simplify the matrix.

Reply:
I can do this in a 2nd PR?  I don't want to change too many things in this one if I can help it (although it might be easier to revert if its all one big change)

Reply:
Oh looking at the code now I think its because its hard to import from files with hyphens in python

Reply:
Yeah I think that's fine. My suggestion is mainly to try to keep things consistent where possible if it's not a big deal. I think we can proceed with this as is but might be something we want to think about standardizing moving forward. Either using underscores or hyphens, whichever is easier.